modules/ROOT/pages/_partials/data-sync/server-offline-and-conflict.adoc

```diff
@@ -1,45 +1,24 @@
 [id="conflict-resolution-{context}"]
-= Resolving conflicts on the server
 
-Applications that allow users to modify data while offline most likely have to deal with conflicts.
-A *conflict* occurs whenever two or more clients try to modify the same data in between synchronizations.
+= Detecting Conflicts on the Server
 
-*Example:* A user tried to modify a record while they were offline. When they came back online, they discovered that the record was already deleted.
-
-*Conflict resolution* is how the application handles the conflict and ensures the correct data is stored. In most cases, the way conflicts are detected and resolved are incredibly specific to an application and the underlying data storage.
-
-In a GraphQL server, conflict detection and resolution happens exclusively in *mutations*.
-
-== Resolving Conflicts
-
-At a high level, there is a typical flow to detecting and resolving conflicts.
+At a high level, there is a typical flow to detecting conflicts.
 
 . *A Mutation Occurs* - A client tries to modify or delete an object on the server using a GraphQL mutation.
 . *Read the Object* - The server reads the current object the client is trying to modify from the data source (usually a database).
 . *Conflict Detection* - The server compares the current object with the data sent by the client to see if there was a conflict. The developer can choose _how the comparison_ is done.
```

Comment:
. *Conflict Detection* - The server compares the current object with the data sent by the client to see if there is a conflict. The developer can choose _how_ the comparison is performed